cf-deploy-teardown/tasks/cf-deploy.yml

```diff
@@ -0,0 +1,12 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: splatform/opensuse-kubectl
+
+inputs:
+- name: cf-ci
+- name: s3.scf-kube-yml
+run:
+  path: cf-ci/cf-deploy-teardown/tasks/cf-deploy.sh
```

Comment:
This comment is meant as a comment about the empty file (I couldn't put a comment on an empty file):
- cf-deploy-teardown/tasks/cf-teardown.yml

Why's this here?